go.mod

```diff
@@ -49,19 +49,19 @@ require (
 	golang.org/x/time v0.0.0-20220224211638-0e9765cccd65
 	gopkg.in/inf.v0 v0.9.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.23.8
-	k8s.io/apiextensions-apiserver v0.23.8
-	k8s.io/apimachinery v0.23.8
+	k8s.io/api v0.24.0
```

Comment:
Probably we don't expect to upgrade K8s API right now from 0.23.8 -> 0.24.0. This should not be merged yet.

@astefanutti WDYT?